src/main/kotlin/no/skatteetaten/aurora/boober/utils/Instants.kt

```diff
@@ -0,0 +1,12 @@
+package no.skatteetaten.aurora.boober.utils
+
+import java.time.Instant
+
+
+object Instants {
```

Comment:
Denne veldig "global state".
Fordelen er at man unngår noen ekstra dependencies i klassen, bare hent "now".

Alternativet hadde vel vært å ha en et interface med 1 test implementasjon og 1 "prod" implementasjon som blir injecta til de som trenger denne, men det blir kanskje overkill i denne situasjonen?

Reply:
Jeg hadde dette egentlig som en Service som jeg injecta og så mocket jeg det i testen. Etter en prat med Bent Andre gjorde vi om på det slik det er nå. Jeg er med på at vi kan bruke din strategi hvis vi bruker klokke noe mer, men akkurat nå synes jeg denne er helt grei.

Reply:
Jeg er enig med deg @jarlehansen , at dette er "global state", og som @bjartek nevner så var det i utgangspunktet slik som du skisserer i ditt alternativ - som vil være go-to-metoden å adressere lignende problemer på. Men, etter min vurdering er det ikke så mange ulemper i dette tilfellet å ha en slik "overridebar global metode", selv om jeg ikke ville bruke denne teknikken i særlig mange tilfeller.

Det som egentlig hadde vært nyttig var plattformstøtte for å override now - noe som jeg faktisk tror ville endt opp med en løsning lignende den vi har implementert nå.